app/lib/services.rb

```diff
@@ -31,9 +31,7 @@ def self.registries
     Registries::BaseRegistries.new
   end
 
-  if Rails.configuration.feature_flag_govuk_accounts
-    def self.accounts_api
-      Plek.find("account-manager")
-    end
+  def self.accounts_api
```

Comment:
Ah interesting, fair!

Reply:
Relic from the Env.fetch days, woops - good catch

Reply:
Yeah, I decided to remove because it was causing some issues with testing and didn't seem like it mattered whether the method were available or not.